packages/dd-trace/src/plugins/util/dd_pkg.js

```diff
@@ -0,0 +1,30 @@
+// similar to packages/dd-trace/src/pkg.js, only this always returns the
+// dd-trace's package (and not the version of the require.main file, for example)
+const fs = require('fs')
+const path = require('path')
+
+function findUp (name, cwd) {
+  let directory = path.resolve(cwd)
+  const { root } = path.parse(directory)
+
+  while (true) {
+    const current = path.resolve(directory, name)
+
+    if (fs.existsSync(current)) return current
+    if (directory === root) return
+
+    directory = path.dirname(directory)
+  }
+}
+
+function getDDTracePkg () {
+  const filePath = findUp('package.json', '.')
```

Comment:
`pkg` is not reliable for finding `dd-trace`'s version: if for example `mocha` is being used a test framework, `require.main.filename` will refer to `mocha`, so we'll get `mocha` version.

Reply:
The version of dd-trace is available from `lib/version.js`.